lib/find.js

```diff
@@ -102,7 +102,13 @@ function _find(tree, _options) {
   var options, root;
 
   if (arguments.length === 1) {
-    var decomposition = decompose(tree);
+    var decomposition;
+
+    if (typeof tree === 'object') {
+      decomposition = decompose(tree.inputTree);
```

Comment:
hmm, should we.

1. support that convention, an handle this recursive?
2. is their another mechanism to accomplish this.